README.md

```diff
@@ -1,47 +1,94 @@
 # code-server
 
-`code-server` is [VS Code](https://github.com/Microsoft/vscode) running on a
-remote server, accessible through the browser.
+Run [VS Code](https://github.com/Microsoft/vscode) on any machine anywhere and access it through the browser.
 
-Try it out:
+- **Code anywhere:** Code on your Chromebook, tablet, and laptop with a
+  consistent dev environment. Develop on a Linux machine and pick up from any
+  device with a web browser.
+- **Server-powered:** Take advantage of large cloud servers to speed up tests, compilations, downloads, and more.
+  Preserve battery life when you're on the go since all intensive tasks runs on your server.
+  Make use of a spare computer you have lying around and turn it into a full development environment.
+
+![Example gif](./doc/assets/code-server.gif)
+
+## Getting started
+
+For a proper setup and walkthrough, please see [./doc/guide.md](./doc/guide.md).
+
+### Debian, Ubuntu
 
 ```bash
-docker run -it -p 127.0.0.1:8080:8080 -v "$PWD:/home/coder/project" -u "$(id -u):$(id -g)" codercom/code-server:latest
+curl -sSOL https://github.com/cdr/code-server/releases/download/3.3.0/code-server_3.3.0_amd64.deb
+sudo dpkg -i code-server_3.3.0_amd64.deb
+sudo systemctl --user enable --now code-server
+# Now visit http://127.0.0.1:8080. Your password is in ~/.config/code-server/config.yaml
 ```
 
-- **Code anywhere:** Code on your Chromebook, tablet, and laptop with a
-  consistent dev environment. Develop on a Linux machine and pick up from any
-  device with a web browser.
-- **Server-powered:** Take advantage of large cloud servers to speed up tests,
-  compilations, downloads, and more. Preserve battery life when you're on the go
-  since all intensive computation runs on your server.
+### Fedora, Red Hat, SUSE
 
-![Example gif](/doc/assets/code-server.gif)
+```bash
+curl -sSOL https://github.com/cdr/code-server/releases/download/3.3.0/code-server-3.3.0-amd64.rpm
+sudo dnf install code-server-3.3.0-amd64.rpm
+sudo systemctl --user enable --now code-server
+# Now visit http://127.0.0.1:8080. Your password is in ~/.config/code-server/config.yaml
+```
 
-## Getting Started
+### npm
 
-### Requirements
+```bash
+npm install -g code-server
+code-server
+```
```

Comment:
To match the others should we put the `Now visit` line here?

Reply:
Opted against so that user's would read the output but I'll add for visual consistency.

Reply:
That makes sense.